src/main/java/org/graylog/aws/kinesis/KinesisConsumer.java

```diff
@@ -90,12 +124,39 @@ public void run() {
 
             @Override
             public void initialize(InitializationInput initializationInput) {
-                LOG.info("Initializing Kinesis worker for stream <{}>", kinesisStreamName);
+                LOG.debug("Initializing Kinesis worker for stream <{}>", kinesisStreamName);
+                transport.consumerState = KinesisTransportState.RUNNING;
             }
 
             @Override
             public void processRecords(ProcessRecordsInput processRecordsInput) {
-                LOG.debug("Received {} Kinesis events", processRecordsInput.getRecords().size());
+
+                LOG.info("processRecords called. Received {} Kinesis events", processRecordsInput.getRecords().size() * 1024);
```

Comment:
Should this really be an info log statement or rather a debug one? Also, why are you multiplying the number of records by 1024?

Reply:
Definitely should be debug. The 1024 number is invalid and unneeded. I'll remove.